### ISSUE TITLE
Missing alt props in images

### ISSUE THREAD

**Issue by samajammin** [ethereum/ethereum-org-website#5289]
**Describe the bug**

It appears we have images that are missing `alt` props.

**To Reproduce**

1. Run the app (`yarn start`)
2. Open in browser (e.g. `localhost:80000/en/`)
3. See console error

```
Warning: Failed prop type: The "alt" prop is required in GatsbyImage. If the image is purely presentational then pass an empty string: e.g. alt="". Learn more: https://a11y-style-guide.com/style-guide/section-media.html
```

```
Warning: Failed prop type: The prop `alt` is marked as required in `MainImage`, but its value is `undefined`.
    at MainImage
```

```
Warning: Failed prop type: The prop `alt` is marked as required in `Y`, but its value is `undefined`.
```

**Expected behavior**

All images should have an `alt` prop.

**Screenshots**

![Image 2022-01-31 at 10 49 19 AM](https://user-images.githubusercontent.com/8097623/151854296-edb2a15a-eb40-40db-a2b9-f7024446d33d.jpg)

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by HursitTarcan** [comment]
Can I try to fix this issue?

**Comment by minimalsm** [comment]
Hey @HursitTarcan, thanks for offering to help. For sure you can! 

I've assigned you, please ping here if you need any help on this one.

**Comment by minimalsm** [comment]
Closed by #5340
